crates/sui-cost/Cargo.toml

```diff
@@ -21,6 +21,7 @@ sui-core = { path = "../sui-core" }
 insta = { version = "1.21.1", features = ["redactions", "json"] }
 test-utils = { path = "../test-utils" }
 sui-config = {path = "../sui-config"}
+sui-move-build = { path = "../sui-move-build" }
```

Comment:
Related to my comment in `calibration.rs` I would expect this target to depend on `sui-move-build` and not directly on `move-package`.

Reply:
ty, fixed up--removed the dependency on `move-package` by updating to use the sui `BuildConfig`. eye-balling, there is lingering setup code that we could make consistent at some point, e.g., use `BuildConfig::new_for_testing()` throughout but it needs more massaging.